NuCivic/dkan_starter
# Add seckit module to dkan_starter.

## Description 
Http Strict Transport Security (HSTS) policy enables web applications to enforce web browsers to restrict communication with the server over an encrypted SSL/TLS connection for a set period. Policy is declared via special Strict Transport Security response header. Encrypted connection protects sensitive user and session data from attackers eavesdropping on network connection.
A successful MiTM attack can lead to the compromise of sensitive user data as well as grant unauthorized access to user accounts enabling attackers to perform privileged actions on client’s behalf.

## Solution

@topicus added the seckit module as @acouch proposed.

## QA steps

- update config.yml to set default:seckit:enable to true.
- run ahoy build config
- run ahoy site up
- Open the chrome developer tools and click in the network tab
- Click on the page request and verify the Strict-Transport-Security header is present in the list of reponse headers
